models/ed/tests/test.write.restart.ED2.R

```diff
@@ -0,0 +1,33 @@
+library(PEcAn.ED2)
```

Comment:
I'm going to side with @mccabete on this one -- up to now things in the 'tests' folder have been limited to formal tests called by testthat. Development code / examples have been kept in inst.